static/usr/lib/systemd/system/console-conf@.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Ubuntu Core Firstboot Configuration %I
+After=systemd-user-sessions.service plymouth-quit-wait.service
+After=rc-local.service
+# on core18/core20 the firstboot seeding happens via the core[18].start-snapd service,
+# make sure to start after so that the 'snap' command is available
+After=core18.start-snapd.service core.start-snapd.service
+# on core20 the user may invoke a recovery chooser, make sure the detection
+# service runs before
+After=snapd.recovery-chooser-trigger.service
+IgnoreOnIsolate=yes
```

Comment:
This also needs an explanation or be removed. I do not see why it should not be stopped when shutting down for instance.

Reply:
I didn't change too much of the service definitions wrt to what's in the upstream right here https://github.com/canonical/subiquity/blob/main/debian/console-conf.console-conf%40.service to keep the diff for a cross review small. If that's no needed I can clean this up, though I think it's be preferable to do it after the thing lands, anyways your call.

Reply:
We can mark them as todos and test in a different PR.

Reply:
dropped too